### PR TITLE
[INVE-23376] Prevent errors from invocation test when folder is within Investigate folder

### PR DESCRIPTION
**Pull request by daradermody**
Errors in the issue were being thrown because Typescript _also_ looks at parent directories for types (e.g. `../node_modules/@types`), which was causing conflicts. We can prevent this by setting `typeRoots` explicitly.

Plus some other minor version bumps.

You can test this by running through the tutorial [here](https://docs.siren.io/siren-platform-user-guide/14.0/web-service-development/advanced_tutorial.html), but instead of installing `@sirensolutions/generator-web-service` using `npm install -g`, you checkout this branch and run `npm link`. Then continue with the tutorial and generate the web service within `kibi-internal/plugins` and try out the `node invoke` command.

### CODE REVIEW

**Comment by colmose** [comment]
I'll let @sibisimon review this one. Once it's merged, can you get a build and send it to Manu?

**Review by sibisimon — APPROVED** [review]
LGTM
image

**Pull request merged**
